tools/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+#
+# CMake file for the vital tools
+#
+
+###
+# Add our plugin path to the default list
+include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
+
+###
+# Add path to applets so plugin loader can find them
+kwiver_make_module_path( ${KWIVER_BINARY_DIR} modules/applets )
+kwiver_add_module_path( ${kwiver_module_path_result} )
+
+###
+#
+kwiver_add_library( kwiver_tools
+               kwiver_applet.h
+               kwiver_applet.cxx
+)
+
+###
+#
+kwiver_add_executable( kwiver
+           kwiver_tool_runner.cxx
+)
+
+target_link_libraries( kwiver
+  PRIVATE        vital
+                 vital_vpm
+                 vital_logger
+                 kwiversys
+                 kwiver_tools
+)
+
+###
+# Add applet for config
+kwiver_add_plugin( tool_config_explorer
```

Comment:
Maybe this is just and example, but it seems like it would be better to not have one plugin for each tool.  We could have one plugin for all vital tools, maybe another for all sprokit tools.

Reply:
Can easily add multiple applets to a plugin. I started with tis approach for porting existing tools which may have bad namespace habits.